### ISSUE TITLE
[BITV] 9.2.1.1/3.3 - Dialog "Online status"  - Sighted keyboard users may have trouble selecting status since the affordance of the status controls will make them think the controls are buttons while it is in fact a radio group (which is logically correct) and thus not tabbable - and the focus on the first (selected) element is not visible since it is not differentiated from the visual indication of selected state (see rating in 9.2.4.7 Focus visible). (5)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36981]
<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v3-n3

</details>

- [ ] 9.2.1.1/5.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v5-n2
- [ ] 9.2.1.1/7.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v7-n4
- [ ] 9.2.1.1/8.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v8-n2
- [ ] 9.2.1.1/10.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v10-n3

**Comment by Fenn-CS** [comment]
Hello @JuliaKirschenheuter and @michaelnissenbaum so the issue here is clear. It would also be nice if more information can be provided about what the correct accessibility fix is.

**Comment by JuliaKirschenheuter** [comment]
![Peek 2023-06-29 10-33](https://github.com/nextcloud/server/assets/6078378/cc27b59a-b685-45dc-8da3-a6dfc371622b)

**Comment by JuliaKirschenheuter** [comment]
Hi @Fenn-CS,
Thanks for question, indeed was not very clear.
radio button are implemented correct. The thing is that an active state don't differentiate with focus state. I will check it designers.

**Comment by JuliaKirschenheuter** [comment]
https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio/
